src/types/drive.ts

```diff
@@ -131,13 +131,16 @@ export interface SqliteDriveItemRow {
   icon_id: number | null;
   is_folder: boolean;
   name: string;
+  plain_name: string | null;
```

Comment:
Why this can be null? Not all files contains plain_name? Or is because current SQlite stored files doesn't have one?

Reply:
Correct, I have left the possibility of null for files that are already stored and have no plain_name to avoid unexpected crashes.